org.flightgear.FlightGear.json

```diff
@@ -0,0 +1,193 @@
+{
+	"id":"org.flightgear.FlightGear",
+	"runtime":"org.kde.Platform",
+	"runtime-version":"5.10",
+	"sdk":"org.kde.Sdk",
+	"command":"flightgear.sh",
+	"rename-icon":"flightgear",
+	"finish-args":[
+		"--share=ipc",
+		"--socket=x11",
+		"--socket=wayland",
```

Comment:
Does it actually support wayland?

Reply:
Apparently not sufficiently: https://forum.flightgear.org/viewtopic.php?f=37&t=32169